MainWindow.cpp

```diff
@@ -201,22 +201,33 @@ void MainWindow::on_actionNext_triggered() { ui->mdiArea->activateNextSubWindow(
 void MainWindow::on_actionPrevious_triggered() { ui->mdiArea->activatePreviousSubWindow(); }
 
 void MainWindow::on_actionDocumentation_triggered() {
-  QUrl documentationURL("https://enigma-dev.org/docs/Wiki/Main_Page", QUrl::TolerantMode);
+  QSettings settings;
+  const QString &documentationURI =

```

Comment:
Confused as to why this object needs to be constructed for every function; might just inline that construction.

Reply:
That's the idiom for QSettings, it's designed that way. The default constructor constructs it with the application name and developer you specified to QApplication.
http://doc.qt.io/qt-5/qcoreapplication.html#applicationName-prop

What do you mean by inline it?

Reply:
I inlined it in at least one way: 7ec936cdc4199a724fdc8e91e02074defd4b8074

Not sure if it's what you meant, but it eliminates duplication which makes it less error-prone.